android/build.gradle

```diff
@@ -44,6 +44,12 @@ android {
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
         targetCompatibility JavaVersion.VERSION_1_8
+    }
+        kotlinOptions {
```

Comment:
The indentation is not correct here. Could you fix that please?

Reply:
Done